src/commands/init.js

```diff
@@ -248,22 +259,21 @@ function interactiveMenu(directory, list) {
     });
 }
 
-function checkFolder(force = false, directoryName = '') {
+function checkFolder(force = false, directory = '') {
     return new Promise((resolve) => {
-        const directoryPath = path.join(process.cwd(), directoryName);
+        const directoryPath = getAbsolutePath(directory || process.cwd());
```

Comment:
I like it, but I don't think it will work. In order to keep the behavior in my added test cases I would have to
```
getAbsolutePath(directoryName || directory || process.cwd())
```
as neither `directoryName` or `directory` are mandatory (`getAbsolutePath` returns undefined if the `filepath` is not defined.)

This is the way I think it should work:

| # | directoryName | directory | directoryPath     |
| --- | ------------------ | ------------- | -------------------- |
| 1 | undefined        | undefined | /cwd                 |
| 2 | name               | undefined | /cwd/name       |
| 3 | /name              | undefined | /name              |
| 4 | undefined        | directory   | /cwd/directory  |
| 5 | undefined        | /directory  | /directory         |
| 6 | name               | directory   | /cwd/directory/name |
| 7 | name               | /directory   | /directory/name |

6 and 7 will not work, so to support that we could do something like:
```
getAbsolutePath(path.join(directory, directoryName));
```

Do you think it would work?